templates/navigation-bar-new.twig

```diff
@@ -72,69 +84,41 @@
 				</span>
 			</button>
 		</div>
-		<form id="search_form" class="form nav-search-form" action="{{ data_nav_bar.home_url }}">
-			{% set search_label = __( 'Search', 'planet4-master-theme' ) %}
-			<input id="{{ search_input_id }}"
-				type="search"
-				name="s"
-				class="form-control"
-				placeholder="{{ search_label }}"
-				value="{{ data_nav_bar.search_query|e('wp_kses_post')|raw }}"
-				aria-label="{{ __( 'Search input', 'planet4-master-theme' ) }}"/>
-			<input id="orderby" type="hidden" name="orderby" value="{{ selected_sort ?? default_sort }}"/>
-			<button class="nav-search-btn"
-				aria-label="{{ __( 'Press return/enter or click to search', 'planet4-master-theme' ) }}"
-				type="submit"
+	</div>
```

Comment:
This closing tag probably inadvertently moved from [here](https://github.com/greenpeace/planet4-master-theme/pull/1628/files?diff=split&w=1#diff-3ad24a02d9e63bde53c8195a030484bfb200dac551677c03584386ba3394c92aL137) (easier to see with whitespace ignored).

As a result the search form has a different container and becomes significantly wider.